Tests/BackendIntegrationTests/LoadShedderIntegrationTests.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  LoadShedderIntegrationTests.swift
+//
+//  Created by Nacho Soto on 3/21/23.
+
+import Nimble
+@testable import RevenueCat
+import StoreKit
+import XCTest
+
+class LoadShedderStoreKit2IntegrationTests: LoadShedderStoreKit1IntegrationTests {
+
+    override class var storeKit2Setting: StoreKit2Setting {
+        return .enabledForCompatibleDevices
+    }
+
+}
+
+class LoadShedderStoreKit1IntegrationTests: BaseStoreKitIntegrationTests {
+
+    override var apiKey: String { return Constants.loadShedderApiKey }
+
+    override class var storeKit2Setting: StoreKit2Setting {
+        return .disabled
+    }
+
+    private var logger: TestLogHandler!
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+
+        self.logger = TestLogHandler()
+    }
+
+    func testCanGetOfferings() async throws {
+        let receivedOfferings = try await Purchases.shared.offerings()
+        expect(receivedOfferings.all).toNot(beEmpty())
+    }
+
+    func testGetOfferingsRedirectsToLoadShedder() async throws {
+        _ = try await Purchases.shared.offerings()
+
+        try self.verifyRedirection(path: .getOfferings(appUserID: Purchases.shared.appUserID))
+    }
+
+    func testCanPurchasePackage() async throws {
+        try await self.purchaseMonthlyOffering()
+    }
+
+    @available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
+    func testProductEntitlementMapping() async throws {
+        try AvailabilityChecks.iOS15APIAvailableOrSkipTest()
+
+        let result = try await Purchases.shared.productEntitlementMapping()
+        expect(result.entitlementsByProduct).toNot(beEmpty())
+    }
+
+    @available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
+    func testProductEntitlementMappingRedirectsToLoadShedder() async throws {
```

Comment:
@aboedo will probably need to remove this, sounds like the server will actually redirect internally..